### DISCUSSION TITLE
Cancelled Copilot. Oops. Now no option to subscribe anywhere.

### DISCUSSION THREAD

**Discussion by markcrandall** [community/community#111174] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I cancelled my copilot free trial and then went to the billing page to subscribe. But the billing page says: 

> You cancelled your free one-time 30 days trial and won't be charged for Copilot. Your access to GitHub Copilot expires in 7 days. [Read billing documentation](https://docs.github.com/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot) for more details.

**and gives me no option to subscribe**, which I would like to do.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @markcrandall , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by Akash1134** [comment]
Hi @markcrandall ! The best place to go for answers to account-specific or billing related questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
